addons/purchase/models/purchase.py

```diff
@@ -476,6 +564,8 @@ class PurchaseOrderLine(models.Model):
              "  - Stock Moves: the quantity comes from confirmed pickings\n")
     qty_received = fields.Float("Received Qty", compute='_compute_qty_received', inverse='_inverse_qty_received', compute_sudo=True, store=True, digits='Product Unit of Measure')
     qty_received_manual = fields.Float("Manual Received Qty", digits='Product Unit of Measure', copy=False)
+    qty_to_invoice = fields.Float(compute='_compute_to_invoice_qty', string='To Invoice Quantity', store=True, readonly=True,
```

Comment:
do we need it to be a field? do we need it to be stored?

Reply:
I guess it's the same situation as that for qty_invoiced, it's not necessary but we still make it a stored field. I won't insist to do that, but in sale, we have the same filed. It's cool to have consistent design, right?

Reply:
alright 👍